prymer/api/probe.py

```diff
@@ -0,0 +1,22 @@
+from dataclasses import dataclass
+
+from fgpyo.util.metric import Metric
+
+from prymer.api.primer import Primer
+
+
+@dataclass(frozen=True, init=True, kw_only=True, slots=True)
+class Probe(Primer, Metric["Probe"]):
```

Comment:
**issue** Per discussion on the working group call today, we are going to add these fields directly to `Primer` (and maybe rename the class to `Oligo` or similar)

Reply:
I should have documented this because I just bumped up against why I defaulted them both to `None` in the first place-- if you specify the default weights for either task, then Primer3 uses those weights as part of your scoring function, and it will complain if the corresponding params are not there, because it has no delta-off-optimal to score.